lang/cs/Org.Apache.REEF.Network/StreamingCodec/StreamWithPosition.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Org.Apache.REEF.Network.StreamingCodec
+{
+    public class StreamWithPosition
```

Comment:
This needs some explanation. Wouldn't we always read from / write to the current position of the stream? When would we want to move it?